src/app/screens/Onboard/ConnectRaspiBlitz/index.tsx

```diff
@@ -0,0 +1,231 @@
+        if (addResult.accountId) {
+          await utils.call("selectAccount", {
+            id: addResult.accountId,
+          });
+          navigate("/test-connection");
+        }
+      } else {
+        alert(`
+          Connection failed. Are your RaspiBlitz credentials correct? \n\n(${validation.error})`);
+      }
+    } catch (e) {
+      console.error(e);
+      let message =
+        "Connection failed. Are your RaspiBlitz credentials correct?";
+      if (e instanceof Error) {
+        message += `\n\n${e.message}`;
+      }
+      alert(message);
+    }
+    setLoading(false);
+  }
+
+  function dropHandler(event: React.DragEvent<HTMLDivElement>) {
+    event.preventDefault();
+    if (
+      event.dataTransfer.items &&
+      event.dataTransfer.items[0].kind === "file"
+    ) {
+      const file = event.dataTransfer.items[0].getAsFile();
+      if (file) {
+        const extension = file.name.split(".").pop();
+        if (extension === "macaroon") readFile(file);
+      }
+    }
+    if (isDragging) setDragging(false);
+  }
+
+  function readFile(file: File) {
+    const reader = new FileReader();
+    reader.onload = function (evt) {
+      if (evt.target?.result) {
+        const macaroon = utils.bytesToHexString(
+          new Uint8Array(evt.target.result as ArrayBuffer)
+        );
+        if (macaroon) {
+          setFormData({
+            ...formData,
+            macaroon,
+          });
+        }
+      }
+    };
+    reader.readAsArrayBuffer(file);
+  }
+
+  function dragOverHandler(event: React.DragEvent<HTMLDivElement>) {
+    event.preventDefault();
+    if (!isDragging) setDragging(true);
+  }
+
+  function dragLeaveHandler(event: React.DragEvent<HTMLDivElement>) {
+    if (isDragging) setDragging(false);
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="relative mt-14 lg:flex space-x-8 bg-white dark:bg-gray-800 px-12 py-10">
+        <div className="lg:w-1/2">
+          <h1 className="text-2xl font-bold dark:text-white">
+            Connect to your RaspiBlitz node
+          </h1>
+          <p className="text-gray-500 mt-6 dark:text-gray-400">
+            You need your node URL and a macaroon with read and send permissions
+            (e.g. admin.macaroon).
+            <br />
+            <br />
+            <b>SSH</b> into your <b>RaspiBlitz</b>.<br />
+            Run the command <b>sudo cat /mnt/hdd/tor/lndrest8080/hostname</b>.
+            <br />
+            Copy your <b>.onion</b> address.
+            <br />
+            Add <b>https://</b> at the beginning and <b>:800</b> at the end.
+            <br />
+            Paste the URL in the input below.
+          </p>
+          <div className="w-4/5">
+            <div className="mt-6">
+              <TextField
+                id="url"
+                label="REST API host and port"
+                placeholder="https://your-node:8080"
+                onChange={handleChange}
```

Comment:
can you sync that with the latest textfield from lnd? 
there is an additional `pattern` validation. 

(that's why code needs to be DRY :D)